boot/zephyr/Kconfig

```diff
@@ -453,7 +511,7 @@ config BOOT_SERIAL_DETECT_PORT
 	string "GPIO device to trigger serial recovery mode"
 	default GPIO_0 if SOC_FAMILY_NRF
 	help
-	  Zephyr GPIO device which contains the pin used to trigger
```

Comment:
Any particular reason to change this? Both 'which' and 'that' would be fine here.

Reply:
I was [asked to change the phrasing](https://github.com/mcu-tools/mcuboot/pull/882#discussion_r526215347) in the new config variable, so I changed the original one to match.

Reply:
Ok. That's fine. I guess I've been around too much non-American English. The that for restrictive clauses is an American thing.